### DISCUSSION TITLE
Topic ideas for TPAC 2024

### DISCUSSION THREAD

**Discussion by spectranaut** [w3c/aria#2283] [General]
It looks like TPAC 2025 in Anaheim will be well attended!

Let's use this discussion page to propose discussion topics for the F2F meeting. Add a comment below with any topic you think we should discuss. Additionally, you can add the tag "F2FCandidate" to any existing issue that you want discussed at TPAC. 

We will record relevant information for the working group and draft schedules on this wikipage: https://github.com/w3c/aria/wiki/TPAC-2024-ARIA-Meetings

**Comment by aleventhal** [comment]
1. Prohibited name repair (prototype available in Chromium)
    Subtopic: what to do if tabindex=0. Seems better to have a minimum role for these cases instead of prohibiting name.

1.  Status update on new HTML and CSS features that are likely ship soon, with a big a11y impact:
Web platform feature improvements, e.g.:
  - [Popover (MDN docs)](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API)
  - [CSS anchor positioning (MDN docs)](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_anchor_positioning)
  - [Customizable select (explainer)](https://open-ui.org/components/customizableselect/)
  - [popover=hint (explainer)](https://open-ui.org/components/popover-hint.research.explainer/)
  - command/commandfor attributes ([explainer](https://open-ui.org/components/invokers.explainer/), [spec PR](https://github.com/whatwg/html/pull/9841), [spec issue](https://github.com/whatwg/html/issues/9625)) 
  - [interesttarget attribute (explainer)](https://open-ui.org/components/interest-invokers.explainer/)
  - [details/summary a11y improvement](https://docs.google.com/document/d/1IO5m67Ngw_J1OmNBdzGhB7Rd7B-9vcuQWtrGaeovsJE/edit#heading=h.hbp1zf7jgf7w)
  - The CSS reading-flow property ([CSS spec](https://drafts.csswg.org/css-display-4/#reading-flow), [WHATWG proposal](https://github.com/whatwg/html/issues/10407), [display:contents issue](https://github.com/whatwg/html/issues/10533), [position:absolute issue](https://github.com/whatwg/html/issues/10539))
  - [::scroll-marker, scrolltarget](https://github.com/flackr/carousel/tree/main/scroll-marker)

**Reply by aardrian** [reply]
I came here to raise CSS reading order because I think we need to spend some time with the CSSWG folks.

**Reply by aleventhal** [reply]
I have a good handle on that one and have been providing input. We have a prototype in Chrome. Today the WG agreed to handle display:contents cases, but, some of the weirder/harder edge cases don't make a lot of sense to bend over backwards for. It comes down to this: you can have intervening display:contents ancestor between the reading container and reading items, but only DOM siblings can be reordered relative to each other. Anything else causes problems.

**Reply by aardrian** [reply]
Are you saying we don't need to spend time with the CSSWG folks for CSS reading order?

**Reply by aleventhal** [reply]
We probably really don't. I don't think there will be anything worth using everyone's time on in a big meeting.

**Reply by aleventhal** [reply]
Maybe we can try having me a give a summary before TPAC on a call and see if anyone sees a blocker that needs wider discussion. It's a good feature for a11y and the only friction was how to deal with weird cases without breaking browser a11y impls.

**Comment by smhigley** [comment]
I'd like to propose a session on interactive lists to discuss a more specific proposal that I will write sometime in the next week and then link here :D

**Comment by chrishtr** [comment]
Proposing two sessions:

1. snap-to-activate: https://github.com/w3c/aria/issues/2659
2. menu element: https://github.com/w3c/aria/issues/2658
